### PR TITLE
BOOM!

### PR DESCRIPTION
**Pull request by trayharrington**

### CODE REVIEW

File: lib/phone_number_formatter.rb [hunk 1]
```diff
@@ -2,5 +2,14 @@
 # returns a properly formatted phone number.
 
 def format_phone_number(phone_number_str)
-  
+  digits = []
+  input = phone_number_str.split(//)
+  input.each do |x|
+    if x == "0" || x == "1" || x == "2" || x == "3" || x == "4" || x == "5" || x == "6" || x == "7" || x == "8" || x == "9"
```
Comment (sumeetjain) [inline lib/phone_number_formatter.rb]: If you had an array `["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]`, you could check whether or not `x` was in that array. That'd be a cleaner way of accomplishing this long conditional :) Do some research and see if you can find a solution like that.

**Comment by sumeetjain** [comment]
:+1:

**Comment by trayharrington** [comment]
I'm so mad that I didn't think of that! Soooo much cleaner.

**Comment by sumeetjain** [comment]
:) It happens to the best of us! Yours was still a good algorithm though.

**Pull request closed without merging**